### ISSUE TITLE
make check-loca is failing on CentOS 7

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/gingerbase#132]
```
/bin/pep8 --version
1.4.6
/bin/pep8 --filename '*.py,*.py.in' --exclude="*config.py,*i18n.py,*tests/test_config.py" .
./tests/test_netinfo.py:120:16: E121 continuation line indentation is not a multiple of four
./tests/test_netinfo.py:122:16: E121 continuation line indentation is not a multiple of four
./tests/test_netinfo.py:133:16: E121 continuation line indentation is not a multiple of four
./tests/test_netinfo.py:135:16: E121 continuation line indentation is not a multiple of four
make[3]: *** [check-local] Error 1
make[3]: Leaving directory `/root/wok/src/wok/plugins/gingerbase'
make[2]: *** [check-am] Error 2
make[2]: Leaving directory `/root/wok/src/wok/plugins/gingerbase'
make[1]: *** [check-recursive] Error 1
make[1]: Leaving directory `/root/wok/src/wok/plugins/gingerbase'
make: *** [check] Error 2
```

**Comment by biancafc** [comment]
Following error happening in Fedora 24:

``` javascript
Makefile:492: recipe for target 'check-local' failed
make[4]: *** [check-local] Error 1
make[4]: Leaving directory '/home/bcarvalho/Documents/kimchi/fedora-24-testes/gingerbase/tests'
Makefile:364: recipe for target 'check-am' failed
make[3]: *** [check-am] Error 2
make[3]: Leaving directory '/home/bcarvalho/Documents/kimchi/fedora-24-testes/gingerbase/tests'
Makefile:366: recipe for target 'check' failed
make[2]: *** [check] Error 2
make[2]: Leaving directory '/home/bcarvalho/Documents/kimchi/fedora-24-testes/gingerbase/tests'
Makefile:604: recipe for target 'check-recursive' failed
make[1]: *** [check-recursive] Error 1
make[1]: Leaving directory '/home/bcarvalho/Documents/kimchi/fedora-24-testes/gingerbase'
Makefile:894: recipe for target 'check' failed
make: *** [check] Error 2
```

**Comment by alinefm** [comment]
Hi @biancafc 

Could you provide the full error details? From your log, I can't get what caused the errors

**Comment by alinefm** [comment]
Original issue fixed by c13abf8e6f56dd7873f7ff6f3e61b66f8f360958
